src/config.rs

```diff
@@ -1026,6 +1027,7 @@ url = "redis://user:passwd@1.2.3.4:6379/1"
 bucket = "name"
 region = "us-east-2"
 endpoint = "s3-us-east-1.amazonaws.com"
+use_ssl = true
```

Comment:
what was the default before your changes ?

Reply:
It's also `true` in this test: https://github.com/mozilla/sccache/blob/41e408d3e22d3c5a03cc622c93e9348ef80df140/src/config.rs#L1004